extension/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "vspmd",
```

Comment:
I think this should be `PreMiD pmd` as it's project bound to not cause any conflicts?